client/templates/userprofile/settings.less

```diff
@@ -13,4 +13,9 @@
 .password_success {
   background-color: darkgreen;
   color: white;
+}
+
+#bio-error {
```

Comment:
Bootstrap has a class for error text. Use existing classes, rather than creating CSS for similar effects.